x-pack/plugins/secops/public/components/timeline/data_providers/facet_text.test.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
```

Comment:
I think that a shallow should do it too here. I do not know if we want to do it like that but I kind of like to see if we are using mount vs shallow vs render so I can already see what I can expect in the test. Make the test more readable.

Reply:
if we want to do it we should apply shallow or mount or render in our test correctly so we can detect what we are doing. My two cents!!! ;)

Reply:
I changed the call from `mount` to `shallow` for this test as requested, but there are a number of reasons why I prefer `mount` to `shallow`.

1) `shallow` often requires the caller to invoke `.dive()` multiple times to get to the correct level in the `DOM` before selectors passed to `.find()` can find their node. In contrast, you can call `wrapper.find()` directly when you use `mount`.
2) Because `shallow` does not render children, calling `.text()` on the wrapper returns a string containing text like `<pure(Component) />` instead of automatically returning the text node rendered by the child component. This makes writing assertions awkward compared to `mount`.
3) Tests that use `mount` are closer to "functional" browser-based tests than tests that use `shallow`, because they exercise the integration between components